addons/website_slides/models/slide_question.py

```diff
@@ -45,6 +47,11 @@ def _compute_statistics(self):
             question.attempts_avg = stats.get('attempts_count', 0) / stats.get('attempts_unique', 1) if stats else 0
             question.done_count = stats.get('done_count', 0) if stats else 0
 
+    @api.depends('answer_ids.is_correct')
```

Comment:
I think that this will not be triggered if new answers are added or removed, so that you also need `'answer_ids'`; this is also necessary to make sure that your constraint doesn't use stale values either.
You could write a small unit test to test your feature and (dis?)prove this.